### DISCUSSION TITLE
ScriptReference/EditorGUIUtility.IconContent

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#271] [Notes]
# ScriptReference/EditorGUIUtility.IconContent

https://docs.unity3d.com/ScriptReference/EditorGUIUtility.IconContent.html

**Comment by docscomments** [comment]
**Comment by `@AnomalousUnderdog`:**  

[This Gist](https://gist.github.com/MattRix/c1f7840ae2419d8eb2ec0695448d4321) is a list of names of built-in icons in the Unity Editor. You can use those in the fist parameter of this method.

**Reply by EvanC4** [reply]
Even better is this site that let's you see the icons along with the names.
https://github.com/halak/unity-editor-icons
